g3doc/quick_reference.md

```diff
@@ -604,6 +604,13 @@ All other ops in this section are only available if `HWY_TARGET != HWY_SCALAR`:
     <code>Vec&lt;D&gt; **WidenMulPairwiseAdd**(D d, V a, V b,)</code>: widens `a`
     and `b` to `TFromD<D>` and computes `a[2*i+1]*b[2*i+1] + a[2*i+0]*b[2*i+0]`.
 
+*   `V`: `u8`, `VU`: `Vec<RebindToUnsigned<DFromV<V>>>`,
```

Comment:
I think V is i8 here. Let's also rename V to VI and D to DI to make this more obvious?
Maybe also in the parameter names, such as au/bi or a_u and b_i?